sympy/assumptions/ask.py

```diff
@@ -25,1124 +26,213 @@ class AssumptionKeys:
 
     @memoize_property
     def hermitian(self):
-        """
-        Hermitian predicate.
-
-        ``ask(Q.hermitian(x))`` is true iff ``x`` belongs to the set of
-        Hermitian operators.
-
-        References
-        ==========
-
-        .. [1] http://mathworld.wolfram.com/HermitianOperator.html
-
```

Comment:
I like the way you have moved these docstrings - it means that `help(Q.hermitian)` gives something useful, which currently it doesn't.